egs2/swbd/asr1/RESULTS.md

```diff
@@ -0,0 +1,25 @@
+<!-- Generated by scripts/utils/show_asr_result.sh -->
+# RESULTS
+## Environments
+- date: `Fri May 14 07:43:22 UTC 2021`
+- model link: https://drive.google.com/drive/folders/18jJyFxSS9nhS51H8xWkMiDybGKrKPVEV?usp=sharing
+- python version: `3.8.8 (default, Apr 13 2021, 19:58:26)  [GCC 7.3.0]`
+- espnet version: `espnet 0.9.9`
+- pytorch version: `pytorch 1.8.0+cu111`
+- Git hash: `64f026d35013e9f0058bcdeab86eb28fed48ed4b`
+  - Commit date: `Fri May 7 09:31:16 2021 +0000`
+
+## asr_train_asr_cformer5_raw_bpe2000_sp
+### WER
+
+```
+exp_sp/train_nodup_sp_pytorch_train_pytorch_conformer_lr5_specaug_resume/decode_eval2000_model.last10.avg.best_decode_train_transformer_lm_pytorch_swbd+fisher_bpe2000/scoring/hyp.callhm.ctm.filt.sys
```

Comment:
Could you double-check that the number of reference words is exactly the same as espnet1 or Kaldi results?

Reply:
I have checked it and it's correct.